### ISSUE TITLE
Can only extract features of two short videos continuously

### ISSUE THREAD

**Issue by 953585130** [facebook/C3D#404]
I put 20 videos selected from ucf101 continuously in the input file. After running the code, I can only extract the first two. From the third output folder, it is blank, but the command window does not report an error.
I0808 21:51:46.354730  2736 net.cpp:165] This network produces output accuracy
I0808 21:51:46.354746  2736 net.cpp:183] Collecting Learning Rate and Weight Decay.
I0808 21:51:46.354759  2736 net.cpp:176] Network initialization done.
I0808 21:51:46.354773  2736 net.cpp:177] Memory required for Data 3391973648
E0808 21:51:48.620146  2736 extract_image_features.cpp:81] Extracting features for 1 batches
E0808 21:51:49.865541  2736 extract_image_features.cpp:117] Extracted features of 30 images.
E0808 21:51:49.865579  2736 extract_image_features.cpp:121] Successfully extracted 30 features!
